src/pages/lib-v1/q/platform/[platform].mdx

```diff
@@ -33,3 +33,27 @@ import android1 from '/src/fragments/lib-v1/android.mdx';
 import flutter2 from '/src/fragments/lib-v1/flutter.mdx';
 
 <Fragments fragments={{ flutter: flutter2 }} />
+
+<InlineFilter filters={['js']}>
+
+## Amplify JavaScript
+
+<Callout warning>
+
+Welcome to the developer preview of **Amplify JavaScript v6**. This preview introduces functional interfaces, typescript improvements and improved NextJS SSR support  for Authentication, Analytics, and Storage.
```

Comment:
TypeScript is incorrectly written. Next.js is inconsistent as well across many files. Recommend find + replace.

Reply:
also a double whitespace after "support"